index.js

```diff
@@ -1,14 +1,62 @@
+<<<<<<< HEAD
+const Discord = require('discord.js');
+const client = new Discord.Client();
+const config = require("./config.json");
+const prefix = config.prefix;
+client.config = config;
+
+const Enmap = require("enmap");
+const fs = require("fs");
+client.commands = new Enmap();
+fs.readdir("./commands/", (err, files) => {
+  if(err)
+    return console.error(err);
+
+  files.forEach(file => {
+    if (!file.endsWith(".js"))
+      return;
+
+    let props = require(`./commands/${file}`);
+    let commandName = file.split(".")[0];
+
+    console.log(`Attempting to load command ${commandName}`);
+    client.commands.set(commandName, props);
+  });
+});
+=======
 const { Client, Attachment } = require('discord.js');
 const https = require('https');
 const client = new Client();
 const prefix = 'hack!';
+>>>>>>> 2c30eaa6be2f0d6231ccbf53865807d389093040
 
 client.on('ready', () => console.log('Bot has logged in!'));
 
 client.on('message', async (msg) => {
   if(msg.author.bot || !msg.content.startsWith(prefix)) return;
   let args = msg.content.split(' ');
   const command = args.shift().slice(prefix.length);
+<<<<<<< HEAD
```

Comment:
more merge conflicts

Reply:
also the merge conflicts are coming from the fact that i changed some things from the ground up. please read the code next time ^^